src/appleseed/renderer/modeling/object/meshobjectoperations.h

```diff
@@ -47,6 +48,9 @@ APPLESEED_DLLSYMBOL void compute_smooth_vertex_normals(MeshObject& object);
 // The mesh object must have texture coordinates.
 APPLESEED_DLLSYMBOL void compute_smooth_vertex_tangents(MeshObject& object);
 
+// Compute a hash for a mesh object.
+APPLESEED_DLLSYMBOL void compute_signature(const MeshObject& object, foundation::MurmurHash& hash);
```

Comment:
Output arguments usually come first in our codebase:

    void compute_signature(foundation::MurmurHash& hash, const MeshObject& object);